<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UusiTiedot</class>
 <widget class="QWidget" name="UusiTiedot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nimi</string>
     </property>
     <property name="buddy">
      <cstring>nimiEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nimiEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Osoite</string>
     </property>
     <property name="buddy">
      <cstring>osoiteEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="osoiteEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Pankkitili (IBAN)</string>
     </property>
     <property name="buddy">
      <cstring>tiliLine</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="tiliLine">
     <property name="text">
      <string>FI</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="muotoLabel">
     <property name="text">
      <string>Yritysmuoto</string>
     </property>
     <property name="buddy">
      <cstring>muotoList</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QListWidget" name="muotoList"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Tilikartan laajuus</string>
     </property>
     <property name="buddy">
      <cstring>laajuusList</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QListWidget" name="laajuusList"/>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>80</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y-tunnus</string>
     </property>
     <property name="buddy">
      <cstring>ytunnusEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="ytunnusEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Kotipaikka</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="kotipaikkaEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nimiEdit</tabstop>
  <tabstop>ytunnusEdit</tabstop>
  <tabstop>osoiteEdit</tabstop>
  <tabstop>kotipaikkaEdit</tabstop>
  <tabstop>tiliLine</tabstop>
  <tabstop>muotoList</tabstop>
  <tabstop>laajuusList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
